sbt-mode-hydra.el

```diff
@@ -28,6 +28,10 @@
 (require 'sbt-mode-buffer)
 (require 'sbt-mode)
 
+(defvar ansi-color-regexp)
```

Comment:
Isn't this the wrong file for this change. Shouldn't it be in sbt-mode-comint?

Reply:
I'm not familiar with the project. I just tried to use it, but the mode wasn't working so I added the fix to the file that contained the offending variable. I believe this was the only file that used `ansi-color-regexp`, but `sbt-mode-comint` certainly sounds like a more appropriate place

Reply:
I could resubmit it to be defined in the other file if you like. As it is, on newer versions of emacs the sbt comint wont function correctly

Reply:
Please do. What do you mean won't function correctly?

Reply:
I mean that the process-filter throws a variable undefined error